api/server/middleware/user_agent.go

```diff
@@ -16,6 +17,7 @@ func NewUserAgentMiddleware(versionCheck string) Middleware {
 
 	return func(handler httputils.APIFunc) httputils.APIFunc {
 		return func(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+			ctx = dockerversion.WithUserAgent(ctx, r.Header.Get("User-Agent"))
```

Comment:
The middleware package needs to be docker packages free, we're still in the process, and this is introducing exactly what we're trying to remove, so, we're not going to do this.